src/main/java/ch/heigvd/pro/b04/moderators/Moderator.java

```diff
@@ -41,8 +41,17 @@
   @Exclude
   private Set<Poll> pollSet;
 
+  /**
+   * add a new Poll {@link Poll} to the polls of this moderator.
+   *
+   * @param newPoll poll to add
+   */
   public void addPoll(Poll newPoll) {
     newPoll.getIdPoll().setIdxModerator(this);
-    this.pollSet = Stream.of(newPoll).collect(Collectors.toSet());
+    if (pollSet == null) {
+      this.pollSet = Stream.of(newPoll).collect(Collectors.toSet());
```

Comment:
Do we need to create a Stream and then collect it, rather than a simple Set with no elements in it ?